VTUL/data-repo
# Update to latest Rails 4 and fix vulnerable gems

**Update to latest Rails 4 and fix vulnerable gems**
* * *

# What does this Pull Request do?

Currently, Rails 4 receives only security updates and, even then, only to the most recent release of Rails 4 (4.2.10). This change upgrades Rails 4 to the latest supported version. It also updates the following gems to address these current known security vulnerabilities:

- rack: CVE-2018-16471
- loofah: CVE-2018-16468
- rubyzip: CVE-2018-1000544
- sprockets: CVE-2018-3760
- actionview: CVE-2016-6316
- activerecord: CVE-2016-6317

# What's the changes?

Edited `Gemfile` to change the `rails` gem version to be the latest for Rails 4. Also removed the version pins on the `sprockets` and `sprockets-rails` gems because they are no longer applicable.

Updated the vulnerable gems listed above via `bundle update rails rack loofah rubyzip sprockets actionview activerecord`.

# How should this be tested?

Deploy this branch of `VTUL/data-repo` and check that the application still works.

# Interested parties
@VTUL/dld-dev
